src/main/java/com/wise23/chariteed/controller/AuthController.java

```diff
@@ -0,0 +1,49 @@
+package com.wise23.chariteed.controller;
+
+import com.wise23.chariteed.model.User;
+import com.wise23.chariteed.service.UserService;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Controller;
+import org.springframework.ui.Model;
+import org.springframework.validation.BindingResult;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestMethod;
+
+import javax.validation.Valid;
+import java.util.List;
+
+@Controller
+public class AuthController {
+    @Autowired
+    UserService userService;
+
+    @RequestMapping(value = {"/login"}, method = RequestMethod.GET)
+    public String login(){
+        return "auth/login";
+    }
+
+    @RequestMapping(value = {"/register"}, method = RequestMethod.GET)
+    public String register(Model model){
+        model.addAttribute("user", new User());
+        return "auth/register";
+    }
+
+    @RequestMapping(value = {"/register"}, method = RequestMethod.POST)
+    public String registerUser(Model model, @Valid User user, BindingResult bindingResult){
+        if(bindingResult.hasErrors()){
+            model.addAttribute("successMessage", "User registered successfully!");
+            model.addAttribute("bindingResult", bindingResult);
+            return "auth/register";
+        }
+        List<Object> userPresentObj = userService.isUserPresent(user);
+        if((Boolean) userPresentObj.get(0)){
+            model.addAttribute("successMessage", userPresentObj.get(1));
```

Comment:
If u want to return a pair of different objects, Use Pair class:
https://docs.oracle.com/javase/9/docs/api/javafx/util/Pair.html

Reply:
`isUserPresent()` does now only return a boolean value. So this comment doesn't apply anymore